source/style-guide/style/code-examples.txt

```diff
@@ -99,44 +101,53 @@ Create a VM Running a Docker Host
       To see how to connect Docker to this machine, run:
       docker-machine env test
 
-#. Run docker-machine ls again to see the VM that you created.
+#. Run the following command again to see the VM that you created.
```

Comment:
This wording feels a bit strange to me. I feel like the previous wording might have been slightly clearer. If we want to go this direction, maybe omit "again" ?